src/main.ts

```diff
@@ -415,10 +408,15 @@ export async function createPipeline(config: Config): Promise<string> {
     core.debug(`Sending pipeline: ${util.inspect(pipeline)}`)
     //TODO: Define and replace different placeholders: e.g. for values, content folders (goss, jmeter), etc.
 
+    const expiresAfter = moment()
+      .add(config.pipelineDuration * 1000, "s")
+      .format("[Date:] ddd, D MMM YYYY HH:mm:ss z")
```

Comment:
I don't think this format is right. The `Date` part is not part of a date. A date in RFC9110 looks more like `Tue, 22 Nov 2022 12:25:13 GMT`.

Reply:
done